content/blog/2022-03-31-training-and-saving-models-on-a-dedicated-runner.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Training and saving models with CML on a dedicated runner
+date: 2022-03-31
+description: Hello World
+descriptionLong: |
+  Some long
+  multiline
+  text (supports _basic_ [Markdown](link))
+picture: 2015-05-01/post-image.jpeg
+pictureComment: Some _Comment_ (supports _basic_ [Markdown](link))
+author: rob_dewit
+# commentsUrl: https://discuss.dvc.org/t/february-22-community-gems/1078
+tags:
+  - CML
+  - DVC
+  - DVC Remotes
+  - Git
+  - Pipelines
+---
+
+# Training and saving models with CML on a dedicated runner
+
+When you first develop a machine learning model, you will probably do so on your
+local machine. You can easily change algorithms, parameters, and input data
+right in your text editor, notebook, or terminal. But what happens when you
+already have a model deployed and want to run multiple experiments? Or if you
+want to deploy a new version daily?
+
+[CML (Continuous Machine Learning)](https://cml.dev/) helps you with this. It is
+an open-source library for implementing continuous integration and delivery
+(CI/CD) in machine learning projects. This way we can define a pipeline to train
+a model and keep track of various versions.
+
+> Imagine we work for the fraud detection team of a webshop and we want to build
+> a model to detect fraudulent orders. One week fraudsters might be trying to
+> order gaming consoles, while the next they might go all-in on diapers. In that
```

Comment:
Minor: sounds like something the bank or payment processor would try to catch. Not sure how the seller could tell the difference or what we mean by fraudulent here.

Reply:
Worked in a team that did this for a few teams, so I could probably give a presentation on that if you want. 😜

Valid point, though. I'll expand upon it a little bit to give better context.

Reply:
You can signup to give a [bash seminar](https://www.notion.so/iterative/4e2f8421bada44e59d3cf818b25372dc?v=ed8d3b1b7d134a5aa1b6c978b4964f33) 😼